package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "playground",
+  "name": "choreography",
   "version": "1.0.0",
-  "description": "",
-  "main": "topology.js",
+  "description": "Pipeline visualizer for Backstage",
+  "main": "dist/index.js",
```

Comment:
Need to make sure this still works in standalone mode if that's a feature we want

Reply:
I'll try checking this out locally and see

Reply:
Still works locally for me. We can send it